server/server.go

```diff
@@ -1343,6 +1365,34 @@ func (s *Server) Start() {
 		s.checkResolvePreloads()
 	}
 
+	// In operator mode, when the account resolver depends on an external system and
+	// the system account is the bootstrapping account, start fetching it
+	if ar := s.accResolver; len(opts.TrustedOperators) == 1 && ar != nil &&
```

Comment:
What happens with multiple operators configured?

Reply:
Currently multiple operators are not documented.
The problem with multiple accounts is to figure out which issuer to pick.
The current workaround is to specify the system account as resolver_preload.
Picking the first operator can be done, but seems like a hack that we avoid in other places as well. (reading options from the operator)

Given that this is an uncommon situation with a workaround I solved for the common case only.
Would you go for the first operator?

Reply:
Design was to always support multiple operators so want to make sure we think through that in this context.

Reply:
Independent of this feature. As far as I can tell, we have no official notion of my operator (operating this server) vs operators who's accounts I trust also. 
Introducing it as new option or documenting that it's the first one probably goes beyond the scope of this review.
Would you agree?

Reply:
There are multiple operator tests AFAIK. We do support it at the lowest level inside the server.

Reply:
Yes, but as we obtain values from the operator JWT such as (system_account name, resolver url and now operator subject) it becomes increasingly necessary to differentiate between the operator of this server (from which these values are read) and operators that are trusted (only).

So far we only extract values when there is only one operator. 
In these other features we default to: No shortcuts. This has to be separately configured.
This feature here is not any different and has the same workaround.

We should discuss a resolution but probably outside of this review.
Including @kozlovic so he's aware of the general issue.

Possible resolutions are:
Define the first jwt to be the jwt of the operator of this server.
Only allow one jwt and have the other operators just be keys.

Probably supporting multiple resolver has to be considered as well.